### ISSUE TITLE
Incompatibility between Ruins of Symbaroum module and Token Action Hud

### ISSUE THREAD

**Issue by dirusulixes** [Drental/fvtt-tokenactionhud#186]
fvtt 9.255
[symbaroum5ecore](https://github.com/FLPub/symbaroum5ecore) 1.0.1
[tokenactionhud](https://github.com/Drental/fvtt-tokenactionhud) 2.2.6

When trying to access the action HUD for a character using the Symbaroum 5e Character sheet symbaroum5ecore defines, HUD is not rendered and this error is thrown:

```
Uncaught (in promise) TypeError: Cannot create property 'slotsAvailable' on number '1'
[Detected 1 package: token-action-hud]
    at bundle.min.js:1:48959
    at Array.forEach (<anonymous>)
    at ActionHandler5e._categoriseSpells (bundle.min.js:1:48760)
    at ActionHandler5e._buildSpellsCategory (bundle.min.js:1:48197)
    at ActionHandler5e._buildCategories (bundle.min.js:1:44688)
    at ActionHandler5e._buildSingleTokenList (bundle.min.js:1:44525)
    at ActionHandler5e.doBuildActionList (bundle.min.js:1:44223)
    at ActionHandler5e.buildActionList (bundle.min.js:1:29750)
    at TokenActionHUD.updateHud (bundle.min.js:1:15688)
```

symbaroum5ecore modifies the DND5e spell system to account for the lack of spell slots in Ruins of Symbaroum. That seems to be the issue when  tokenactionhud tries to render spell information. ~~Deselecting the Display Spell Information option in settings is a workaround to let the HUD render, but loses useful information in the process.~~ (upon further testing, it doesn't seem to workaround much).

It would be great if the two modules could be made fully compatible. Even greater would be for tokenactionhud to render the spell info specific to symbaroum5ecore: no spell slots, and no prepared vs. not prepared, but whether a spell is _favoured_ or not.

**Comment by dirusulixes** [comment]
Actually after more testing, even with the display spell information setting off, some characters trigger the error. For example Zanna from the SRD starter heroes pack.

**Comment by Larkinabout** [comment]
Fixed error in release 3.0.14.

If still required, please open a new issue regarding the better compatibility between Token Action HUD and the Ruins of Symbaroum module.
